lldb/test/API/functionalities/data-formatter/data-formatter-stl/libcxx-simulators/categories

```diff
@@ -0,0 +1 @@
+libc++
```

Comment:
We don't want this, as one of the biggest advantages of these tests is that they don't require an actual libc++ library. Given that, it may not even be very useful to run them as a part of the libc++ suite, but if you really do want to run them here, then i think we ought to create a separate category for this.

Reply:
Yea that's fair. We still include headers in those tests to get type_traits etc.

But yea they shouldn't be libc++ dependent (at least not on the layout of the types).